packages/expo-updates/src/Updates.types.ts

```diff
@@ -22,7 +22,37 @@ export enum UpdateEventType {
   ERROR = 'error',
 }
 
-type UpdateCheckResultRollBackToEmbedded = {
+enum UpdateCheckResultNotAvailableReason {
+  /**
+   * No update manifest or rollback directive received from the update server.
+   */
+  UPDATE_NOT_AVAILABLE_ON_SERVER = 'updateNotAvailableOnServer',
```

Comment:
```suggestion
  NO_UPDATE_AVAILABLE_ON_SERVER = 'noUpdateAvailableOnServer',
```
(to me, "no update available" implies that no update exists on the server, while "update not available" implies that one exists but is not available to this client. even though both could technically be true, it's probably better to default to the former)